src/helpers/coolParameters.js

```diff
@@ -32,7 +32,7 @@ const getUIDefaults = () => {
 
 	const systemDarkMode = window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches
 	// FIXME For now we need to access parent here as the public page loaded for collabora doesn't have the user theme
-	const nextcloudDarkMode = parent.document.body.dataset.themeDark === '' || parent.document.body.dataset.themeDarkHighcontrast === ''
+	const nextcloudDarkMode = parent.document.body.dataset.themeDark === '' || parent.document.body.dataset.themeDarkHighcontrast === '' || (parent.document.body.dataset.themes === '' && systemDarkMode)
```

Comment:
```suggestion
	... (parent.document.body.dataset.themes === '' && systemDarkMode)
```

I cannot see why this would be empty, can you provide some more clear steps to replicate this issue? Is this something on an older Nextcloud release?

Reply:
How I reproduced it:
1. My system is in dark mode
2. created a new user
3. new user should have dark nextcloud theme by default
4. just now open any doc